README.md

```diff
@@ -105,9 +106,8 @@ To test the project, execute the following command:
 
```

Comment:
- [ ] Hey @Nelitaa I hope you are doing well :+1: Well you project is great but i cannot see any commit from your partner side :fearful: Kindly ask your partner to make commits in this pull request because this project is a pair-programming project :thinking: